Front/src/app/models/shipmentViewModel.ts

```diff
@@ -0,0 +1,12 @@
+export class ShipmentViewModel {
+    shipmentid?: number;
+    customerid: number = 0;
+    employyeid: number = 0;
```

Comment:
Same spelling error. 
```suggestion
    employeeid: number = 0;
```

Reply:
solved